app/src/components/fields/DateTimeFields.tsx

```diff
@@ -120,8 +136,8 @@ export const DateTimeFields = <FormikPropsType,>(props: IDateTimeFieldsProps<For
                 name: timeName,
                 required: timeRequired,
                 variant: 'outlined',
-                error: get(touched, timeName) && Boolean(get(errors, timeName)),
-                helperText: get(touched, timeName) && get(errors, timeName),
+                error: get(touched, timeName) && (timeError || Boolean(get(errors, timeName))),
+                helperText: get(touched, timeName) && (timeHelperText || get(errors, timeName)),
```

Comment:
same here

Reply:
We actually never use dateHelperText anywhere, so I might just remove it